modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/api/lifecycle/EditAPIAndCheckUpdatedInformationTestCase.java

```diff
@@ -62,7 +62,7 @@
     public void initialize() throws APIManagerIntegrationTestException, XPathExpressionException,
                                     MalformedURLException {
         super.init();
-        apiEndPointUrl = getGatewayURLHttp() + API_END_POINT_POSTFIX_URL;
+        apiEndPointUrl = backEndServerUrl.getWebAppURLHttp()() + API_END_POINT_POSTFIX_URL;
```

Comment:
This gives compilation error. Please fix this

Reply:
This is already fixed and merged